### DISCUSSION TITLE
直接拉去编译错误

### DISCUSSION THREAD

**Discussion by dd-bpir3** [immortalwrt/immortalwrt#1309] [Q&A]
git clone https://github.com/immortalwrt/immortalwrt
编译时候提示
WARNING: Makefile 'package/feeds/telephony/freeswitch/Makefile' has a dependency on 'libpcre', which does not exist
make[1]: *** [package/feeds/packages/shadowsocks-libev/compile] Error 1
但是git clone -b openwrt-23.05 https://github.com/immortalwrt/immortalwrt.git
编译没问题

**Comment by lonelysoul** [comment]
你在config里过滤下passwall，是不是有关于它的。如果是，你去make menuconfig，把passwall里的和libev相关的取消试试看？

**Reply by dd-bpir3** [reply]
拉去23.05源码passwall可以编译的

**Comment by xuexijiaoben** [comment]
the first problem
sed -i 's/+libpcre/+libpcre2/g' package/feeds/telephony/freeswitch/Makefile

the second i met too, i don't know how to fix

**Reply by dd-bpir3** [reply]
Thank you

**Comment by xuexijiaoben** [comment]
shadowsocks-libev failed

找到这个错误的解释了，编译的时候先去掉它吧
